_locales/de/messages.json

```diff
@@ -200,7 +204,7 @@
         "description": "Erklärung zum Ersetzen von Text in Editoren und Textfeldern"
     },
     "replace_editors_p1": {
-        "message": "Um Text in Texteditoren oder Eingabefeldern zu ersetzen, sollten Sie das Kontrollkästchen <code>Nur Eingabefelder</code> aktivieren. Andernfalls wird der Suchtext im gesamten HTML ersetzt, was möglicherweise die Seite beschädigen kann. Um das Ändern von versteckten Eingabefeldern zu vermeiden, sollten Sie auch das Kontrollkästchen <code>Nur sichtbare Elemente</code> aktivieren.",
```

Comment:
Why did this change? Same question for all translation changes.

Reply:
The original functionality and the functionality of the new extension are slightly different.
So I updated the help section and I changed this.